commons/src/message-transporters/disconnect_reason.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+export enum DisconnectReason {
+  USER_NOT_PERMITTED = 100
```

Comment:
You use the enum value as `code` integer for the `close` method of a WebSocket. As per the RFC defining WebSockets, the private-use space for custom codes is between 4000 and 4999.

https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4.2

Reply:
fixed